app/controllers/publishers/vacancies_controller.rb

```diff
@@ -59,8 +61,22 @@ def summary
     @feedback_form = Publishers::JobListing::FeedbackForm.new
   end
 
+  def job_applications; end
```

Comment:
You don't need this now. As long as the template exists, rails will know what to do